benchmark/Fixtures/Data.php

```diff
@@ -17,4 +21,18 @@ public static function readJsonFile(string $path): array
     {
         return json_decode(file_get_contents($path), true, 512, JSON_THROW_ON_ERROR);
     }
+
+    /**
+     * Generates an in-memory stream of the given size.
+     *
+     * @return resource
+     */
+    public static function getStream(int $size)
+    {
+        $stream = fopen('php://memory', 'w+');
+        fwrite($stream, str_repeat("\0", $size));
```

Comment:
> The dataset, designated GRIDFS_LARGE (disk file 'gridfs_large.bin'), consists of a single file containing about 50 MB of random data.

I don't need to commit a 50MB file full of NULL characters from the spec. I can generate it on-demand in-memory.